src/scenic/core/regions.py

```diff
@@ -2397,11 +2397,21 @@ class PathRegion(Region):
     Args:
         points: A list of points defining a single polyline.
         polylines: A list of list of points, defining multiple polylines.
+        orientation:
```

Comment:
Need documentation here (which should mention which orientation is used if this argument is not provided, as well as how you can disable the default orientation entirely by passing `None`).